test/podmontest/insips.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin
```

Comment:
File name with current standard naming should be "insps.sh"

Reply:
Renamed the file to insps.sh